index.js

```diff
@@ -1,36 +1,39 @@
-// @flow
 'use strict';
 const tempDir = require('temp-dir');
 const tempy = require('tempy');
 const path = require('path');
 const fsExtra = require('fs-extra');
-const onExit = require('signal-exit');
+const { onExit } = require('signal-exit');
 const globby = require('globby');
 
-/*::
-type Opts = {
-  glob?: string | Array<string>,
-  root?: string,
-  cleanup?: boolean,
-};
-*/
-
-function fixturez(cwd /*: string */, opts /*: Opts */ /*:: = {} */) {
-  opts = opts || {};
+/**
+ * @typedef {Object} Opts
+ * @property {string | ReadonlyArray<string>} [glob]
+ * @property {string} [root]
+ * @property {boolean} [cleanup]
+ */
+
+/**
+ * @param {string} cwd
+ * @param {Opts} [opts]
+ */
+function fixturez(cwd, opts = {}) {
   let glob = opts.glob || '{fixtures,__fixtures__}/*';
   let autoCleanup = typeof opts.cleanup === 'boolean' ? opts.cleanup : true;
   let rootDir = opts.root || '/';
 
-  function find(name /*: string */) /*: string */ {
+  /**
+   * @param {string} name
+   * @returns {string}
+   */
+  function find(name) {
     let search = cwd;
     let match;
 
     do {
       let paths = globby.sync(glob, {
```

Comment:
⚠️ this doesn't work right now. I can fix this but I want to check the appetite for this change first - before I spend too much time on this.